lecture-notes/1-cli.md

```diff
@@ -1955,7 +2003,12 @@ run your script using `./myscript.sh`. But what shell is running this file? We
 will find out more about this later (or read ahead to the `#!`
 section).[^shell-complication]
 
-<!-- TODO: Mention that Windows line endings will stop scripts from working -->
+> NOTE: We have had reports of strange shell script behavior from students
```

Comment:
Should this be styled with `> `? Do we do that anywhere else for notes?

Reply:
We have a couple examples of prior art in this file and 4-cor